### PR TITLE
Anomalydetection

### PR DESCRIPTION
**Pull request by carolsworld**
Merge into the main branch

### CODE REVIEW

**Comment by carolsworld** [comment]
Merge into the main branch, thanks.

**Comment by carolsworld** [comment]
Added a folder containing the trained model of anomaly detection

File: TRIST/docker-compose.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ services:
         ipv4_address: 192.168.1.10
     privileged: true
     volumes:
-      - ../:/TRIST
```
Comment (RedClouud) [inline TRIST/docker-compose.yml:15]: Why have the volumes for the containers been changed? The directory above (`../`) contains all the required scripts for the containers to run properly... Unless I'm missing something?

**Comment by RedClouud** [comment]
Apart from the [directory comment](https://github.com/RedClouud/ICS-test-bed-PoC/pull/3#discussion_r1494832218), all looks good. Nice work!

File: TRIST/docker-compose.yml (hunk 1 above)
Comment (carolsworld) [reply TRIST/docker-compose.yml:15]: Please use your original version. i think it is accidentally changed.

Comment (carolsworld) [reply TRIST/docker-compose.yml:15]: the pictures and models i've added are located at my anonmalydetection branch, would you be able to merge all of them into your main branch?

Comment (RedClouud) [reply TRIST/docker-compose.yml:15]: > Please use your original version. i think it is accidentally changed.

Ok, I'll use the original version


> the pictures and models i've added are located at my anonmalydetection branch, would you be able to merge all of them into your main branch?

Ah yep, I can see them now. I missed them out last time by mistake, my bad!

**Pull request merged**